cli/src/format.rs

```diff
@@ -102,54 +102,50 @@ impl Format for ListGroupMembersResponse {
     }
 }
 
+/// Write object fields
+fn write_fields<W: Write>(fields: &[(&str, &str)], writer: &mut W) -> std::io::Result<()> {
+    let max_label_width = fields.iter().map(|f| f.0.len()).max().unwrap_or(0);
+
+    for field in fields {
+        writeln!(writer, "  {}  {}", style(leftpad(field.0, max_label_width)).blue(), field.1)?;
+    }
+    Ok(())
+}
+
 impl Format for Vec<JobDescriptor> {
```

Comment:
There is also an `impl Format for Vec<HistoryJob>` later in this same file, with a different output format. The user will see a different output format when the `--project` option is supplied (on the console with or without `--json` supplied).

This difference was introduced in https://github.com/phylum-dev/cli/pull/947 and may be intended since the discussion then was about how there were/are different endpoints for history, jobs, and groups. Is this still a restriction or can the output formats be consolidated now?

Reply:
This restriction still exists